src/app/identity.service.ts

```diff
@@ -109,7 +110,7 @@ export class IdentityService {
       `${environment.identityApiPrefix}${this.peoplePath}/${person.id}`,
       person,
       this.getAuthHttpOptions(person),
-    );
+    ).pipe(retry(2), delay(2000));
```

Comment:
Not a blocker, but imho worth using this to make thousands easy to read.
```suggestion
    ).pipe(retry(2), delay(2_000));
```